### PR TITLE
Symlink Replacement

### PR DESCRIPTION
**Pull request by sallar**
Replaces symlinks with normal paths.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,6 @@
     "scripts": {
         "build:images": "node resources/prepare-images.js",
         "generate:archive": "node scripts/generate_test_archive.js",
-        "install": "node scripts/link.js",
         "test": "npm run test:lint && npm run test:nodeunit",
```
Comment (perry-mitchell) [inline package.json]: You could also remove `"fs-symlink": "~1.2.1",` maybe..

**Comment by perry-mitchell** [comment]
It actually looks better than I expected.. Though the directories don't serve so much purpose. Perhaps we remove them in the future..

**Comment by sallar** [comment]
@perry-mitchell I removed that dependency too. So if it looks ok to you, maybe we can merge this and then `buttercup-core-web` too, which depends on this. I’ll have to bump it’s `buttercup-core` version up before merging.

**Comment by perry-mitchell** [comment]
I can take a look at it again tomorrow :) But it looks awesome dude, exciting!

**Pull request merged**